api/src/pcapi/scripts/close_listed_offerers/main.py

```diff
@@ -0,0 +1,244 @@
+            "REJECTED": rejected_by_fraud_action,
+            "USER_FIRST_NAME": booking.firstName,
+            "USER_LAST_NAME": booking.lastName,
+            "VENUE_NAME": offer.venue.common_name,
+        },
+    )
+
+
+def send_specific_cancellation_by_pro_to_beneficiary_email(
+    booking: bookings_models.Booking, rejected_by_fraud_action: bool = False
+) -> None:
+    data = get_booking_cancellation_by_pro_to_beneficiary_email_data(booking, rejected_by_fraud_action)
+    mails.send(recipients=[booking.email], data=data)
+
+
+def cancel_booking_on_closed_offerer(booking: bookings_models.Booking, author_id: int | None = None) -> None:
+    bookings_validation.check_booking_can_be_cancelled(booking)
+    try:
+        cancelled = bookings_api._cancel_booking(
+            booking, bookings_models.BookingCancellationReasons.OFFERER_CLOSED, author_id=author_id
+        )
+    except external_bookings_exceptions.ExternalBookingException as exc:
+        logger.info(
+            "API error while cancelling external booking, try to cancel unilaterally",
+            extra={"exc": exc, "booking": booking.id},
+        )
+        cancelled = bookings_api._cancel_booking(
+            booking,
+            bookings_models.BookingCancellationReasons.OFFERER_CLOSED,
+            one_side_cancellation=True,
+            author_id=author_id,
+        )
+    if not cancelled:
+        return
+    logger.info("Cancelled booking on closed offerer", extra={"booking": booking.id})
+    send_specific_cancellation_by_pro_to_beneficiary_email(booking)
+
+
+# Same as in offerers/api.py
+def _cancel_individual_bookings_on_offerer_closure(offerer_id: int, author_id: int | None) -> None:
+    bookings = offerers_api.get_individual_bookings_to_cancel_on_offerer_closure(offerer_id)
+
+    for booking in bookings:
+        with atomic():
+            try:
+                cancel_booking_on_closed_offerer(booking, author_id=author_id)
+            except Exception as exc:
+                mark_transaction_as_invalid()
+                logger.exception(
+                    "Failed to cancel booking when closing offerer",
+                    extra={"exc": exc, "booking_id": booking.id, "offerer_id": offerer_id},
+                )
+
+    db.session.flush()
+
+
+def close_offerer(
+    offerer: offerers_models.Offerer,
+    **action_args: typing.Any,
+) -> None:
+    if offerer.isClosed:
+        raise offerer_exceptions.OffererAlreadyClosedException()
+
+    was_validated = offerer.isValidated
+    offerer.validationStatus = ValidationStatus.CLOSED
+    db.session.add(offerer)
+    history_api.add_action(
+        history_models.ActionType.OFFERER_CLOSED,
+        author=None,
+        offerer=offerer,
+        closure_date=date.today().isoformat(),
+        comment="Entité juridique d'escape games fermée à la suite du ticket https://passculture.atlassian.net/browse/PC-37351",
```

Comment:
Détail : Je pense qu'on peut formuler plus court :
```suggestion
        comment="Entité juridique d'escape games fermée - PC-37351",
```